### PR TITLE
Wakeup animation made

### PR DESCRIPTION
**Pull request by Juhoton**
Wakeup animation done, still need to add script to disable input, just in case. 
Still needs to be tested with other scenes.

### CODE REVIEW

File: debug.log
Comment (viilis) [inline debug.log:1]: delete this

File: .vsconfig
Comment (viilis) [inline .vsconfig:1]: don't push these

**Review by viilis — CHANGES_REQUESTED** [review]

File: ProjectSettings/EditorBuildSettings.asset
Comment (Juhoton) [inline ProjectSettings/EditorBuildSettings.asset:1]: These should be on master?

File: Assets/Prefabs/Movable_Player.prefab
Comment (Juhoton) [inline Assets/Prefabs/Movable_Player.prefab:1]: Not sure about the huge deletion of stuff

Comment (Juhoton) [reply Assets/Prefabs/Movable_Player.prefab:1]: It actually moves them around, so I'm not sure if that is problematic or not.
Lines 3-94 are moved to 231-304

File: debug.log
Comment (Juhoton) [reply debug.log:1]: Deleted

File: .vsconfig
Comment (Juhoton) [reply .vsconfig:1]: Removed

**Review by viilis — APPROVED** [review]

**Pull request merged**